app/models/manageiq/providers/openstack/provider.rb

```diff
@@ -16,6 +16,8 @@ class ManageIQ::Providers::Openstack::Provider < ::Provider
 
   validates :name, :presence => true, :uniqueness => true
 
+  supports :create
```

Comment:
Do we check the Provider class for `create` ?

Reply:
don't think so

Reply:
We can clean this up later